core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -181,17 +199,19 @@ export class DotContainerPropertiesComponent implements OnInit {
     }
 
     /**
-     * Opens modal for confirmation modal.
-     * @return void
+     * Opens modal for confirmation.
+     * @param {number} lastValue
      * @memberof DotContainerPropertiesComponent
      */
-    clearContentConfirmationModal(): void {
+    clearContentConfirmationModal(lastValue: number): void {
         this.dotAlertConfirmService.confirm({
             accept: () => {
                 this.clearContentTypesAndCode();
             },
             reject: () => {
-                //
+                if (this.form.value.maxContentlets <= 0 || !this.form.value.maxContentlets) {
```

Comment:
We should have a `min` attr in tht HTML so is never less than 0.

Reply:
Noted

Reply:
updated